src/main/java/uk/gov/hmcts/sscs/domain/Evidence.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.sscs.domain;
+
+import java.time.LocalDate;
+
+public class Evidence implements Comparable<Evidence> {
```

Comment:
Lombok is useful to get rid of boilerplate code such as get/set toString methos, etc.